addons/website_sale/static/src/scss/website_sale.scss

```diff
@@ -895,7 +895,7 @@ a.no-decoration {
         font-size: $font-size-sm;
     }
 
-    .o_wsale_address_fill {
+    .o_portal_address_fill {
         .col-form-label:not(.label-optional)::after {
```

Comment:
Either it is specific to `website_sale` and should be based on a class specific to `website_sale`, or it's not and then it should be moved to `portal`, but your portal design shouldn't change when you install ecommerce